transports.md

```diff
@@ -17,3 +18,11 @@ TODO: Include details of the websocket message to define a standard usage.
 TODO:
 * Quic
 * Transport binding
+
+#### Queue
```

Comment:
This seems a bit odd to describe it as a transport. It seems like a pattern in how transports are used rather than an actual transport method itself. I agree that this needs to be described within this spec because it's a key function of routing to mobile devices, but let me see if there's a better place for this to live to prevent confusion about what the "transport layer" actually is.

Reply:
Agree it doesn't it one aspect of transports (because it isn't an actual transport.) This section is also used to understand how to process the various endpoint types in a DIDComm service entry of a DID Document. In that aspect, it fits right in.

This is on the list of conversation topics for today's call.